README.adoc

```diff
@@ -54,12 +73,48 @@ You also assert that you are the original author of the work that you are contri
 === Submitting an Issue
 
 We use the {uri-issues}[issue tracker on GitHub] associated with this project to track bugs and features.
+
 Before submitting a bug report or feature request, check to make sure it hasn't already been submitted.
-When submitting a bug report, please include any details that may help reproduce the bug, including the asciidoc content or logs.
 
-=== Submitting a Pull Request
+When submitting a bug report, please include any details that may help reproduce the bug, including the asciidoc content or logs.
 
-TODO
+=== Contributing a Snippet
+
+. Clone https://github.com/anthonny/hubot-asciidoc
+. Open `/hubot-asciidoc/src/asciidoc.coffee`.
+. See <<Snippet>> for the block syntax required.
+. Create the new block.
+. Add a comment to the top of the file, in alphabetical order.
+The comments are displayed when you run `@<botname> help asciidoc`.
+. Push your changes to GitHub and create your PR.
+
+[[Snippet]]
+.Anatomy of a Snippet
+[source,json]
+----
+'comment': <1>
+    title: "Comment" <2>
+    descriptionMoreURL: "http://asciidoctor.org/docs/user-manual/#comments" <3>
+    description: """ <4>
+    ```<5>
+    // A single-line comment.
+    ```
+    """
+----
+<1> The name of the snippet you want other users to specify in the command.
```

Comment:
I omit to say that the name must be in lowercase cause in the code when the bot receive the command i apply a lowercase https://github.com/anthonny/hubot-asciidoc/blob/master/src/asciidoc.coffee#L494